API.md

```diff
@@ -8,13 +8,15 @@ If you discover that some topic related to the behavior of the endpoint operatio
 
 ### Scheme: Basic
 
-_Command:_
+#### Request
+
 ```bash
-curl http://localhost:<port>/irods-http-api/<version>/authenticate \
-    -X POST --user <username>:<password> 
+curl -X POST -u <username>:<password> http://localhost:<port>/irods-http-api/<version>/authenticate
```

Comment:
just confirming...  password is plaintext here?

Reply:
Yes.

If `-u <username>` is used instead, curl will prompt for the password without echoing the input. Mentioning that is out of scope IMO.

Reply:
got it.   carry on.